### PR TITLE
[MIG] 11.0 mig stock picking backorder strategy

### PR DESCRIPTION
**Pull request by mgosai**
#377

### CODE REVIEW

**Comment by mgosai** [comment]
@max3903

**Comment by max3903** [comment]
@mgosai Can you add the maintainers and development status key in the manifest?

@rousseldenis Should he put yourself as the maintainer?

**Comment by rousseldenis** [comment]
@mgosai You have to run the oca-gen-addon-readme tool as there are some incorrect keys in README

**Comment by rousseldenis** [comment]
@max3903 I hope this fits your needs :smile:

**Comment by alexschubert** [comment]
Testing in process.

**Review by max3903 — APPROVED** [review]
@rousseldenis Exactly what we need. Thanks!
@mgosai :+1:

**Review by alexschubert — APPROVED** [review]
Thanks for your contribution!

Functional Testing:
I was able to successfully validate all backorder strategies added by this module.

File: stock_picking_backorder_strategy/README.rst
Comment (rousseldenis) [inline stock_picking_backorder_strategy/README.rst:2]: @mgosai This has been changed manually. Do you plan to run the oca-gen-addon-readme tool before merging or wait for OCA bot to do it ?

Comment (max3903) [reply stock_picking_backorder_strategy/README.rst:2]: @rousseldenis We'll wait for the OCA bot to do it.

**Review by rousseldenis — APPROVED** [review]
Code review

**Pull request merged**
